composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
```

Comment:
`scripts` is root-only: https://getcomposer.org/doc/04-schema.md#scripts

So this needs to be in the Standard Edition.

Also, not everyone can make use of symlinks, so we should probably do it in a similar fashion to https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Composer/ScriptHandler.php + https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php